tests/test_box_utils.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# import random
+import unittest
+
+import numpy as np
+
+# import torch
+from parameterized import parameterized
+
+from monai.data.box_mode import CornerCornerModeTypeA, CornerCornerModeTypeB, CornerSizeMode
```

Comment:
Could you please help enhance the test cases to cover at least all the supported modes?

Thanks.